pkgs/servers/nextcloud/packages/apps/hmr_enabler.nix

```diff
@@ -0,0 +1,38 @@
+{
+  php,
+  fetchFromGitHub,
+  lib,
+}:
+
+php.buildComposerProject2 (finalAttrs: {
```

Comment:
Please don't use the version 2 of the builder, we have some instability issues with it that needs to be investigated. I would not recommend it yet.